tests/functional/test_integration.py

```diff
@@ -6,7 +6,7 @@
 async def test_build_and_deploy(ops_test: OpsTest):
     charm = await ops_test.build_charm(".")
     resources = {
-        "gunicorn-image": "gunicorncharmers/gunicorn-app:20.0.4-20.04_edge"
```

Comment:
Get this from command line arguments instead to prevent coupling the test to the image name tag. You can see an example here:
https://github.com/canonical/indico-operator/blob/main/tests/conftest.py
https://github.com/canonical/indico-operator/blob/main/tests/integration/conftest.py